src/deploy/osp_deployer/director.py

```diff
@@ -1284,16 +1289,27 @@ def setup_sriov_nic_configuration(self):
         sriov_vfs_setting = []
         sriov_map_setting = []
         sriov_pci_passthrough = []
-        physical_network = "physint"
-        for interface in sriov_interfaces:
+        physical_network = ["physint", "physint1"]
+        check = 0
+        for interface in sriov_interfaces:   # Do it for only first two interfaces, if smart nic is enabled. otherwise
             devname = interface
-            mapping = physical_network + ':' + interface
-            sriov_map_setting.append(mapping)
+            if self.settings.enable_smart_nic == True and self.get_sriov_interfaces() == 2: # sriov_interfaces contain only two interfaces
```

Comment:
So this looks line smart NICs will be enabled only if there are 2 SRIOV interfaces.  What if there are 4?  What is the behavior that we want then?  With this code, it looks like no smart NICs will be enabled.